web/src/components/tasks/business-formation/business/IsVeteranNonprofit.tsx

```diff
@@ -35,12 +35,13 @@ export const IsVeteranNonprofit = (): ReactElement => {
           row
         >
           <FormControlLabel
+            className=" margin-right-0"
```

Comment:
```suggestion
            className="margin-right-0"
```
This is such a nit. I'm sorry for even pointing it out.

Reply:
done.